labgrid/target.py

```diff
@@ -349,8 +349,8 @@ def bind_driver(self, client):
                 supplier = suppliers[0]
             if supplier is not None and (requirement, supplier) in bound_req_pairs:
                 raise BindingError(
-                    "duplicate bindings of {} for {} found in target {}".format(
-                        supplier, requirement, self)
+                    "duplicate bindings of {} to {} for {} found in target {}".format(
+                        supplier, supplier_name, requirement, self)
```

Comment:
This should be `name` instead of `supplier_name`, as we're interested in the name of the binding here.

Reply:
Right, fixed.